src/styles/main.scss

```
@@ -1,3 +1,14 @@
-body {
+@import "./utils/variables";
```

```suggestion
@import "./utils/variables";
```
If you have this file here, why do you need it in __product_card.scss?

---

i had troubles with variables and this fixed them((